node/bin/src/block_replay_storage.rs

```diff
@@ -12,8 +12,7 @@ use tokio::pin;
 use tokio::time::{Instant, Sleep};
 use vise::Unit;
 use vise::{Buckets, Histogram, Metrics};
-use zk_ee::system::metadata::BlockMetadataFromOracle;
-use zk_os_forward_system::run::BlockContext;
+use zksync_os_interface::types::BlockContext;
```

Comment:
`BlockContext` has changed, it's a breaking change, so we will need to migrate storage:
- re-sync one of ENs with new image
- rotate main node
- resync other nodes